docs/topics/deployment/manually-with-asgi-server.rst

```diff
@@ -0,0 +1,102 @@
+Manually with ASGI server
+==========
+
+ASGI (Asynchronous Server Gateway Interface) is intended to provide a standard interface between async Python web frameworks like Litestar, and async web servers such as the ones below.
+
+Uvicorn
+-------
+
+`Uvicorn <https://www.uvicorn.org/>`_ is an ASGI server that supports HTTP/1.1 and WebSocket.
+
+Hypercorn
+---------
+
+`Hypercorn <https://hypercorn.readthedocs.io/en/latest/#/>`_ is an ASGI server that was initially part of `Quart <https://pgjones.gitlab.io/quart//>`_, and supports HTTP/1.1, HTTP/2, and WebSocket.
+
+Daphne
+------
+
+`Daphne <https://github.com/django/daphne/>`_ is an ASGI server that was originally developed for `Django Channels <https://channels.readthedocs.io/en/latest/>`_, and supports HTTP/1.1, HTTP/2, and WebSocket.
+
+Granian
+-------
+
+`Granian <https://github.com/emmett-framework/granian/>`_ is a Rust based ASGI server that supports HTTP/1.1, HTTP/2, and WebSocket.
```

Comment:
Lets put these into a tab group as well - note the following suggestions to add though

Reply:
great suggestion! done in https://github.com/litestar-org/litestar/pull/2663/commits/188600113e06a505bdd476235d1f08480ace2616